airflow/__init__.py

```diff
@@ -39,7 +39,7 @@
 
 settings.initialize()
 
-login = None
+login: None = None
```

Comment:
```suggestion
login = None # type: None
```
Unfortunately, we support Python 3.5. The annotations for variables are not supported in this version.